x/auth/ante.go

```diff
@@ -255,7 +269,7 @@ func ensureSufficientMempoolFees(ctx sdk.Context, stdTx StdTx) sdk.Result {
 	if stdTx.Fee.Gas <= 0 {
 		return sdk.ErrInternal(fmt.Sprintf("invalid gas supplied: %d", stdTx.Fee.Gas)).Result()
 	}
-	requiredFees := adjustFeesByGas(ctx.MinimumFees(), stdTx.Fee.Gas)
+	requiredFees := adjustFeesByGas(ctx.MinimumFees(), stdTx.Fee.Gas, params.GasPerUnitCost)
```

Comment:
`GasPerUnitCost` absolutely does not belong in the param store. It is a local variable, not in consensus.

Reply:
Updated.